data/json/items/tool/lighting.json

```diff
@@ -271,6 +271,7 @@
     "type": "TOOL",
     "name": { "str": "glowstick" },
     "description": "This is a small blue light glowstick.  Use it to bend the plastic and break the glass cylinder inside to start the reaction.  It will produce a small amount of light.",
+    "flags": [ "SLEEP_IGNORE" ]
```

Comment:
Actually I think, need to add a flag to `glowstick_lit` item, not to regular `glowstick`

Reply:
Of course. Thank you!